dart/constraint/ContactConstraint.hpp

```diff
@@ -144,6 +144,9 @@ class ContactConstraint : public ConstraintBase
       const dynamics::ShapeNode* shapeNode);
   static double computeSecondaryFrictionCoefficient(
       const dynamics::ShapeNode* shapeNode);
+  static double computeSlipCompliance(const dynamics::ShapeNode* shapeNode);
```

Comment:
Please rename to `computePrimarySlipCompiance()` for the consistency

Reply:
Done in https://github.com/dartsim/dart/pull/1505/commits/5d81fbbc3a61d493f7f4f201657aa482ed169d2c